src/caterpillar_test.py

```diff
@@ -669,5 +678,127 @@ def test_meta_tags(self):
       self.assertIn('<meta content="test233" name="name"', js_file.read())
 
 
+class TestEndToEndConvert(TestCaseWithTempDir):
```

Comment:
This should not be in the same file as the rest of the unit tests. Add a new file called end_to_end_test.py or something.